integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiNewCreateAuxImage40.java

```diff
@@ -0,0 +1,604 @@
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  private static LoggingFacade logger = null;
+  private String domain1Uid = "domain1";
+  private String domain2Uid = "domain2";
+  private String domain3Uid = "domain3";
+  private static String miiAuxiliaryImage1 = MII_AUXILIARY_IMAGE_NAME + "-new1";
+  private static String miiAuxiliaryImage2 = MII_AUXILIARY_IMAGE_NAME + "-new2";
+  private static String miiAuxiliaryImage3 = MII_AUXILIARY_IMAGE_NAME + "-new3";
+  private final int replicaCount = 2;
+  private static String adminSecretName;
+  private static String encryptionSecretName;
+  public static final String ORACLELINUX_TEST_VERSION =
+      Optional.ofNullable(System.getenv("ORACLELINUX_TEST_VERSION")).orElse("7");
+
+  /**
+   * Install Operator.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Creating unique namespace for domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createOcirRepoSecret(domainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    adminSecretName = "weblogic-credentials";
+    createSecretWithUsernamePassword(adminSecretName, domainNamespace, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    encryptionSecretName = "encryptionsecret";
+    createSecretWithUsernamePassword(encryptionSecretName, domainNamespace,
+        ENCRYPION_USERNAME_DEFAULT, ENCRYPION_PASSWORD_DEFAULT);
+
+    // build app
+    assertTrue(buildAppArchive(defaultAppParams()
+            .srcDirList(Collections.singletonList(MII_BASIC_APP_NAME))
+            .appName(MII_BASIC_APP_NAME)),
+        String.format("Failed to create app archive for %s", MII_BASIC_APP_NAME));
+  }
+
+  /**
+   * Create a domain using auxiliary images. Create auxiliary image using default options.
+   * Verify the domain is running and JMS resource is added.
+   */
+  @Test
+  @DisplayName("Test to create domain using createAuxImage with default options")
+  void testCreateDomainUsingAuxImageDefaultOptions() {
+
+    // admin/managed server name here should match with model yaml
+    final String auxiliaryImageVolumeName = "auxiliaryImageVolume1";
```

Comment:
This variable is never used.

Reply:
Fixed in 8efab86, I removed the unused variables.